is-browser-window/is-browser-window.js

```diff
@@ -22,8 +22,10 @@ var globals = require('can-globals/can-globals-instance');
  */
 
 globals.define('isBrowserWindow', function(){
+	var isNode = globals.getKeyValue('isNode');
 	return typeof window !== "undefined" &&
-		typeof document !== "undefined" && typeof SimpleDOM === "undefined";
+		typeof document !== "undefined" &&
+		isNode === false;
```

Comment:
the `isNode` I'm not 100% sure on long-term.  I think NWJS should be considered a browser window.  I'm not sure how we can sniff this out, but `isNode ==== false` is fine for now.  We'll need something later.

Reply:
done-autorender has a change for when running in nwjs. I agree though, using clientWidth or something would be a better solution long-term.

Reply:
Opened an issue to fix this long-term: https://github.com/canjs/can-globals/issues/36